pkgs/tools/graphics/xcftools/default.nix

```diff
@@ -0,0 +1,38 @@
+{stdenv, fetchurl, libpng, perl, gettext }:
+
+stdenv.mkDerivation {
+  name = "xcftools-1.0.7";
+
+  src = fetchurl {
+    url = "http://henning.makholm.net/xcftools/xcftools-1.0.7.tar.gz";
+    sha256 = "19i0x7yhlw6hd2gp013884zchg63yzjdj4hpany011il0n26vgqy";
+  };
+
+  buildInputs = [ libpng perl gettext ];
+
+  patchPhase = ''
+    # Required if building with libpng-1.6, innocuous otherwise
+    substituteInPlace xcf2png.c         \
+      --replace png_voidp_NULL NULL     \
+      --replace png_error_ptr_NULL NULL
+
+    # xcfview needs mailcap and isn't that useful anyway
+    sed -i -e '/BINARIES/s/xcfview//' Makefile.in
+  '';
+
+  meta = {
+    homepage = http://henning.makholm.net/software;
+    description = ''
```

Comment:
Please edit description.
https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes
Use less than 81 symbols, don't mention pkg's name and don't use dot at the end.